pandas/core/strings.py

```diff
@@ -680,12 +680,33 @@ def str_repeat(arr, repeats):
 
     Parameters
     ----------
-    repeats : int or array
+    repeats : int or array of int
         Same value for all (int) or different value per (array)
 
     Returns
     -------
-    repeated : Series/Index of objects
+    Series  or Index of objects
+
+    Examples
+    --------
+
+    >>> s = pd.Series(['a', 'b', 'c'])
+    >>> s
+    0    a
+    1    b
+    2    c
+
+    Single int repeats string in Series
+    >>> s.str.repeat(2)
```

Comment:
I think we need a blank line between the text and the code. Or at least we've been leaving it, and it'd be good to be consistent.

Can you provide the argument as a keyword argument? I think `.repeat(repeats=2)` is more explicit thatn `.repeat(2)` (may be would make sense to call the parameter `.repeat(times=2)` but that's for another issue.